pkg/doc/verifiable/presentation_ldp.go

```diff
@@ -0,0 +1,22 @@
+package verifiable
+
+import (
+	"fmt"
+)
+
+// AddLinkedDataProof appends proof to the Verifiable Presentation.
+func (vp *Presentation) AddLinkedDataProof(context *LinkedDataProofContext) error {
```

Comment:
Similar question as earlier - is there a reason that linked data proofs are being added after instantiation?

Reply:
Yes, if you would like to sign them. This is convenience method for signing VC.